### PR TITLE
fix missing spaces in adjacent strings

### LINKED ISSUE #48090: "No MaterialLocalizations found" exception has a typo
For reasons that don't matter here, I got the following exception:
```
E/flutter (18304): [ERROR:flutter/lib/ui/ui_dart_state.cc(157)] Unhandled Exception: No MaterialLocalizations found.
E/flutter (18304): MyApp widgets require MaterialLocalizations to be provided by a Localizations widget ancestor.
E/flutter (18304): Localizations are used to generate many different messages, labels,and abbreviations which are used by the material library.
E/flutter (18304): To introduce a MaterialLocalizations, either use a MaterialApp at the root of your application to include them automatically, or add a Localization widget with a MaterialLocalizations delegate.
E/flutter (18304): The specific widget that could not find a MaterialLocalizations ancestor was:
E/flutter (18304):   MyApp
E/flutter (18304): The ancestors of this widget were:
E/flutter (18304):   [root]
```

Notice the typo on the third line, `labels,and`.

Ironically, we even hard-coded this typo into the relevant test:
https://github.com/flutter/flutter/blob/dfbd58566818abc63a87d60098e013bc4c05feb3/packages/flutter/test/material/debug_test.dart#L71

This is caused by omitting a space character at the end of one string in a multiline multipart string literal:
https://github.com/flutter/flutter/blob/449f4a6673f6d89609b078eb2b595dee62fd1c79/packages/flutter/lib/src/material/debug.dart#L79

To catch these in the future, we should write a test (or a lint) that looks for adjacent string literals where the last character of the first literal is not whitespace (U+0020 or U+000A).

cc @a14n if this is something that would be easy to lint.

### PR DESCRIPTION
**Pull request by a14n**
## Description

Add missing space in adjacent strings

## Related Issues

Fixes #48090

## Tests

None

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d
Nice!

File: packages/flutter/lib/src/widgets/automatic_keep_alive.dart [hunk 1]
```diff
@@ -145,7 +145,7 @@ class _AutomaticKeepAliveState extends State<AutomaticKeepAlive> {
       assert(() {
         if (!mounted) {
           throw FlutterError(
-            'AutomaticKeepAlive handle triggered after AutomaticKeepAlive was disposed.'
+            'AutomaticKeepAlive handle triggered after AutomaticKeepAlive was disposed. '
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/automatic_keep_alive.dart]: this should actually be a newline (see docs for `FlutterError` constructor)

**Comment by Hixie** [comment]
This is fantastic, thanks @a14n!

File: packages/flutter/lib/src/widgets/automatic_keep_alive.dart (hunk 1 above)
Comment (a14n) [reply packages/flutter/lib/src/widgets/automatic_keep_alive.dart]: Addressed in #49280

**Pull request merged**